include/dkm_utils.hpp

```diff
@@ -9,6 +9,41 @@
 
 namespace dkm {
 
+
+/**
+ * Calculates the Euclidean distance from each point in the given sequence
+ * to given center and writes the results onto the sequence starting
+ * with out.
+ *
+ * @param points Point sequence.
+ * @param out    Beginning of the output distance sequence.
```

Comment:
These comments are now incorrect after the changes to the function parameter list.